elements/rh-button/rh-button.css

```diff
@@ -0,0 +1,490 @@
+.light {
+  /* PRIMARY */
+  --_primary-color: var(--rh-color-text-primary-on-dark, #ffffff);
+  --_primary-background-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_primary-border-color: transparent;
+  --_primary-border-width: var(--rh-border-width-sm, 1px);
+  --_primary-active-color: var(--rh-color-text-primary-on-dark, #ffffff);
+  --_primary-active-background-color: var(--rh-color-interactive-blue-darkest, #004080);
+  --_primary-active-border-width: var(--rh-border-width-sm, 1px);
+  --_primary-focus-color: var(--rh-color-text-primary-on-dark, #ffffff);
+  --_primary-focus-background-color: var(--rh-color-interactive-blue-darkest, #004080);
+  --_primary-focus-border-width: var(--rh-border-width-md, 2px);
+  --_primary-hover-color: var(--rh-color-text-primary-on-dark, #ffffff);
+  --_primary-hover-background-color: var(--rh-color-interactive-blue-darkest, #004080);
+  --_primary-hover-border-width: var(--rh-border-width-sm, 1px);
+
+  /* DANGER */
+  --_danger-color: var(--rh-color-text-primary-on-dark, #ffffff);
+  --_danger-background-color: #c9190b; /* WARNING: non-token value */
```

Comment:
`--_danger-background-color` should be `--rh-color-red-600` for light theme `primary danger` button. Same color works for `secondary danger` button's text and border color.